Tasks/SqlAzureDacpacDeployment/DeploySqlAzure.ps1

```diff
@@ -39,6 +39,30 @@ Import-Module "Microsoft.TeamFoundation.DistributedTask.Task.DevTestLabs"
 # Load all dependent files for execution
 Import-Module ./Utility.ps1 -Force
 
+function Get-SingleFile($files, $pattern)
```

Comment:
instead of using $files use $directoryPath or $filesPath as it is the directory location where we want to find file